projects/mci/test/test_mci_output.c

```diff
@@ -114,6 +116,22 @@ static void prv_setup_motor_can(void) {
   mcp2515_init(&s_motor_can_storage, &mcp2515_settings);
 }
 
+static StatusCode prv_empty_ack_callback(CanMessageId msg_id, uint16_t device, CanAckStatus status,
+                                         uint16_t num_remaining, void *context) {
+  return STATUS_CODE_OK;
+}
+
+static StatusCode prv_set_regen_braking_state(uint8_t state) {
+  CanAckRequest req = { .callback = prv_empty_ack_callback,
+                        .context = NULL,
+                        .expected_bitset =
```

Comment:
It looks like something is wrong with the formatting here.
My guess is to try and add a comma after `CAN_ACK_EXPECTED_DEVICES(SYSTEM_CAN_DEVICE_MOTOR_CONTROLLER)`

Reply:
done